server/repository/src/mock/mod.rs

```diff
@@ -596,7 +596,7 @@ impl MockDataInserts {
         self
     }
 
-    pub fn full_master_list(mut self) -> Self {
+    pub fn full_master_lists(mut self) -> Self {
```

Comment:
The rest are plural, the property it modifies is plural... sorry it touched many files 🫨